### PR TITLE
Add 2 linter filters

### PR DESCRIPTION
**Pull request by rlustin**
Add new linter filters:
- `NoSpaceBeforeCommaFilter` (all locales);
- `SpaceBeforeDoublePonctuationFilter` (fr only).

@Signez, could you please review?

### CODE REVIEW

File: src/linter/mod.rs [hunk 1]
```diff
@@ -52,6 +40,23 @@ impl Linter {
...
+
+    fn active_filters(&self, locale: &str) -> Vec<Box<LinterFilter>> {
+        self.filters()
+            .into_iter()
+            .filter( |filter| filter.locales().is_empty() ||
+                filter.locales().contains(&locale) )
+            .collect()
+    }
+
+    fn filters(&self) -> Vec<Box<LinterFilter>> {
```
Comment (Signez) [inline src/linter/mod.rs]: Oh, you're right. Wayyyyy better way to write it. A shame I didn't spot that in the first place…

File: src/linter/mod.rs [hunk 2]
```diff
@@ -52,6 +40,23 @@ impl Linter {
             Err(warnings)
         }
     }
+
+    fn active_filters(&self, locale: &str) -> Vec<Box<LinterFilter>> {
+        self.filters()
+            .into_iter()
+            .filter( |filter| filter.locales().is_empty() ||
```
Comment (Signez) [inline src/linter/mod.rs]: I'm surprised by the presence of a space _before_ the pipe for the lambda arguments (here: `filter( |filter| …`).

File: src/linter/filters/no_space_before_comma_filter.rs [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        let filter = NoSpaceBeforeCommaFilter {};
+
+        let result = filter.check("Lorsqu’on le lui demande , il répond qu’il se nomme Simbad le marin.");
+
+        assert!(result.is_err());
+
+        let warnings = result.err().unwrap();
+
+        assert_eq!(1, warnings.len());
+        assert_eq!("Please don’t use a space before a comma.",warnings[0].message);
```
Comment (Signez) [inline src/linter/filters/no_space_before_comma_filter.rs]: There is a space missing here before `warnings`.

File: src/linter/mod.rs (hunk 2 above)
Comment (rlustin) [inline src/linter/mod.rs]: Updated to:

```rust
.filter(|filter| filter.locales().is_empty() || filter.locales().contains(&locale))
```

File: src/linter/mod.rs (hunk 1 above)
Comment (rlustin) [inline src/linter/mod.rs]: :)

File: src/linter/filters/no_space_before_comma_filter.rs (hunk 3 above)
Comment (rlustin) [inline src/linter/filters/no_space_before_comma_filter.rs]: :+1:

**Comment by rlustin** [comment]
Thanks for you review. Fixed!

**Pull request merged**